pkg/volume/util/hostutil/hostutil_linux_test.go

```diff
@@ -283,6 +283,7 @@ func TestGetFileType(t *testing.T) {
 
 	for idx, tc := range testCase {
 		path, cleanUpPath, err := tc.setUp()
+		defer os.RemoveAll(cleanUpPath) // RemoveAll can deal with a empty path ""
```

Comment:
This waits until the end of the function, which is ok I guess.

Even if RemoveAll can deal with an empty path, I think it'd be good to only call this if setUp successfully got a path. It's super annoying to have a bug result in a test deleting something it shouldn't...